ReactiveCocoa/NSObject+RACDescription.m

```diff
@@ -48,3 +48,15 @@ - (NSString *)rac_description {
 }
 
 @end
+
+NSString *rac_description(id object) {
+	if ([object respondsToSelector:@selector(rac_description)]) {
```

Comment:
Sounds like a sensible idea, but I'm concerned that the ``-class`` method (used in the ``rac_description`` implementation) is not necessarily implemented by an ``id`` object.